src/components/Chip/Chip.tsx

```diff
@@ -26,13 +26,14 @@ interface ContainerProps {
 const Container = styled.span<ContainerProps>`
   --_accentColor: ${color.inactive};
   --_backgroundColor: ${color.background};
+  --_borderRadius: ${space[32]};
```

Comment:
```suggestion
  --_borderRadius: ${({ squared }) => squared ? space[8] : space[32]};
```

I think this is a bit more clean, then you don't need a custom styled component. You do have to pass the squared prop to the main chip component, but I think this also just makes sense, so all the chip's can be "squared".